youtube_dl/extractor/kick.py

```diff
@@ -0,0 +1,53 @@
+# coding: utf-8
+from __future__ import unicode_literals
+
+from .common import InfoExtractor
+from ..utils import url_or_none, traverse_obj, strip_or_none, float_or_none, parse_iso8601, int_or_none
+
+
+class KickIE(InfoExtractor):
+    _VALID_URL = r'https?://(?:www\.)?kick\.com/video/(?P<id>[0-9a-zA-Z-]+)'
+    _TEST = {
+        'url': 'https://kick.com/video/82a3c11d-7a17-4747-aecb-2e61413eb11f',
+        'md5': 'f052bc1046cd9ca6751925dd12420010',
+        'info_dict': {
+            'id': '82a3c11d-7a17-4747-aecb-2e61413eb11f',
+            'ext': 'mp4',
+            'title': 'Weekly Stake Stream',
+            'uploader': 'Eddie',
+            'thumbnail': r're:^https?://.*\.jpg.*$',
+            'timestamp': 1676890314,
+            'upload_date': '20230220',
+        }
+    }
```

Comment:
Preferred for new extractors -- missed it before.
```suggestion
    _TESTS = [{
        'url': 'https://kick.com/video/82a3c11d-7a17-4747-aecb-2e61413eb11f',
        'md5': 'f052bc1046cd9ca6751925dd12420010',
        'info_dict': {
            'id': '82a3c11d-7a17-4747-aecb-2e61413eb11f',
            'ext': 'mp4',
            'title': 'Weekly Stake Stream',
            'uploader': 'Eddie',
            'thumbnail': r're:^https?://.*\.jpg.*$',
            'timestamp': 1676890314,
            'upload_date': '20230220',
        }
    }]
```